### PR TITLE
Bump deps

### PR DESCRIPTION
**Pull request by kidonng**
~~Rename `isGlobalSearchResults` to `isGlobalSearch`. This `results` does not matter and now it aligns with `isRepoSearch`.~~

While making this I noticed the test is failing, so this PR now just fix the tests.

### CODE REVIEW

**Comment by fregante** [comment]
Technically there’s a /search page without results though 😃

**Comment by kidonng** [comment]
Then let's bump ts-node instead 😀

**Comment by fregante** [comment]
esm-lint does not depend on ts-node. Maybe npm got updated the the output of npm pack changed? It should be the name of the archive generated. It’s then immediately unpacked.

**Comment by kidonng** [comment]
> esm-lint does not depend on ts-node.

There are two issues here. `ts-node` should be bumped.

File: .github/workflows/esm-lint.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
       - run: npm install
       - run: npm run build --if-present
       - run: npm pack --dry-run
-      - run: npm pack --silent 2>/dev/null | xargs -n1 tar -xzf
+      - run: npm pack --silent 2>/dev/null | grep -v '>' | xargs -n1 tar -xzf
```
Comment (kidonng) [inline .github/workflows/esm-lint.yml:26]: Fix upstreamed: https://github.com/fregante/ghatemplates/pull/9

**Comment by fregante** [comment]
Can you bump all deps instead? The tree dependency should not be installed here

**Pull request closed without merging**